### PR TITLE
History of Videos

### PR DESCRIPTION
**Pull request by Jia-A**
In this PR, following features are there : 
 - Clicked videos are sent to History Page.
 - You can remove a particular video from History Page.
 - You can clear the whole history.

### CODE REVIEW

**Review by kavyakunder — APPROVED** [review]
Code is clean and easily understandable.
Looks good to merge!

File: src/context/videoContext.js [hunk 1]
```diff
@@ -55,8 +74,45 @@ const VideoProvider = ({children}) => {
...
+    }
+    }
+
+    const removeHistory = async ( token, _id) =>{
+        try{
+            const response = await removeHistoryVideos(token, _id)
+            videoDispatch({type : "REMOVE_HISTORY", payload : response.history})
+        }
+        catch(error){
+            console.log(error)
```
Comment (Rohitprasad83) [inline src/context/videoContext.js:163]: try using toast, do not keep console.logs

**Review by Rohitprasad83 — APPROVED** [review]
- Code looks good
- Good Naming conventions were used.

File: src/context/videoContext.js (hunk 1 above)
Comment (Jia-A) [reply src/context/videoContext.js:163]: Oh okay, I have yet to learn how react toast works but will change consoles with them for sure.

**Pull request merged**